frontend/src/modules/entitydetails/components/EntityDetails.test.js

```diff
@@ -209,9 +213,11 @@ describe('<EntityDetails>', () => {
     });
 
     it('dispatches the updateStatus action when updateTranslationStatus is called', () => {
-        const [wrapper] = createEntityDetailsWithStore();
+        const [wrapper, store] = createEntityDetailsWithStore();
 
         wrapper.instance().updateTranslationStatus(42, 'fake translation');
+        // Proceed with changes even if unsaved
+        store.dispatch(unsavedchanges.actions.ignore());
```

Comment:
I couldn't explain why this wasn't needed before, any ideas @adngdb?

Reply:
That's weird. Maybe the unsavedchanges state is "polluted" and thus must be ignored before the action is actually called? You could check that by printing the value of `store.getState().unsavedchanges` before this line, to see if it has `exists` as true. As for why that would be, I do not know! Either a precedent test failed to clean the state, or there is something in the setup that triggers the unsavedchanges action.

Reply:
Hmmm I went through the possible code paths again, and the main difference now being how `EntityDetails` is mounted, this also means underlying components are mounted, so the logic under `GenericTranslationForm` will be executed, which in turn fires the `unsavedchanges.actions.update()` action via the `useUpdateUnsavedChanges` hook.

And that brings us now to have the `unsavedchanges.actions.exists` populated to `true`, and because we are invoking `EntityDetail` component's `.updateTranslationStatus()` directly, that will only execute the `history.actions.updateStatus()` action when no unsaved changes exist (as part of the callback function).

Long story short: mounting brings us closer to real user interactions, and the line I added makes sense in this context.